frontend/src/components/Header.js

```diff
@@ -1,59 +1,54 @@
-import React, { useEffect, useState } from 'react'
+import React from 'react'
+import { useNavigate } from 'react-router-dom' // Import useNavigate for navigation
 
 function Header() {
-  const [isLoggedIn, setIsLoggedIn] = useState(false)
-  const [username, setUsername] = useState('')
-  useEffect(() => {
-    const checkLoginStatus = () => {
-      const user = JSON.parse(localStorage.getItem('user'))
-      setIsLoggedIn(!!user)
-      setUsername(user?.name || '')
-    }
-    checkLoginStatus()
-    const handleStorageChange = (event) => {
-      if (event.key === 'user') {
-        checkLoginStatus()
-      }
-    }
-    window.addEventListener('storage', handleStorageChange)
-    return () => {
-      window.removeEventListener('storage', handleStorageChange)
+  const navigate = useNavigate()
```

Comment:
Why did you remove the logged-in logic? This PR is supposed to be only UI changes.

Reply:
see it again i fixed it .